packages/langchain/src/openai/util.ts

```diff
@@ -275,8 +295,8 @@ export function mapLangchainToAiClient(
     top_logprobs: options?.top_logprobs,
     function_call: options?.function_call,
     stop: options?.stop ?? client.stop,
-    functions: options?.functions?.map(mapToolToOpenAiFunction),
-    tools: options?.tools?.map(mapToolToOpenAiTool),
+    functions: options?.functions?.map(f => mapToolToOpenAiFunction(f)),
```

Comment:
Why add f => here?

Reply:
I have to. Because now the map function takes a second param `strict` and ts thinks it should be `index` if not write like this.